src/main/java/com/alexlasota/medicalclinic/service/VisitService.java

```diff
@@ -23,10 +24,18 @@ public class VisitService {
     private final VisitRepository visitRepository;
     private final VisitMapper visitMapper;
 
+    // TC1: W przypadku gdy zostanie wywolana metoda z visitRepo findAll to zostanie zwrocona lista wizyt.
     public List<Visit> getVisits(Pageable pageable) {
         return visitRepository.findAll(pageable).getContent();
     }
 
+    //TC1: W przypadku gdy istnieje doctor o danym id, wizyty nie nachodzą na siebie oraz data start nie jest po dacie end
```

Comment:
Jako pierwsze masz walidacje dotyczące godzin wizyty, powinno to być zawarte w każdym z TC.